packs/data/consumable-effects.db/effect-cognitive-mutagen-greater.json

```diff
@@ -1,8 +1,9 @@
 {
     "_id": "qit1mLbJUyRTYcPU",
     "data": {
+        "badge": null,
         "description": {
-            "value": "<p>Granted by @Compendium[pf2e.equipment-srd.Cognitive Mutagen (Greater)]{Cognitive Mutagen (Greater)}</p>\n<p><strong>Implemented effects</strong></p>\n<ul>\n<li>+3 item bonus to Arcana, Crafting, Occultism and Society checks</li>\n<li>+3 item bonus to all Recall Knowledge checks</li>\n<li>-2 penalty to to Athletics and Acrobatics checks</li>\n<li>-2 penalty to weapon and unarmed attacks</li>\n</ul>\n<p><strong>Unimplemented effects</strong></p>\n<ul>\n<li>+3 item bonus to Lore checks</li>\n<li>Treat Recall Knowledge critical failures as failures</li>\n<li>You can carry 2 less Bulk than normal before becoming encumbered</li>\n<li>The maximum Bulk you can carry is reduced by 4</li>\n</ul>"
+            "value": "<p>Granted by @Compendium[pf2e.equipment-srd.Cognitive Mutagen (Greater)]{Cognitive Mutagen (Greater)}</p>\n<p><strong>Implemented effects</strong></p>\n<ul>\n<li>+3 item bonus to Arcana, Crafting, Occultism and Society checks</li>\n<li>+3 item bonus to all Recall Knowledge checks</li>\n<li>-2 penalty to to Athletics and Acrobatics checks</li>\n<li>-2 penalty to weapon and unarmed attacks</li>\n<li>+3 item bonus to Lore checks</li>\n<li>Treat Recall Knowledge critical failures as failures</li>\n</ul>\n<p><strong>Unimplemented effects</strong></p>\n<ul>\n<li>You can carry 2 less Bulk than normal before becoming encumbered</li>\n<li>The maximum Bulk you can carry is reduced by 4</li>\n</ul>"
```

Comment:
Any specific reasoning? Admitting I really like it as a user since it reminds me what I need to manually do for my character, and I like it as a dev because it gives me a better at-a-glance idea of what could be worth looking at again in the future.

Reply:
Noting here for documentation: The short conversation on discord showed that other people find this useful. Can this PR be approved?